crates/solvers/src/domain/solver/dex/solver.rs

```diff
@@ -1,109 +0,0 @@
-//! A simple solver that matches orders directly with swaps from the external
```

Comment:
`mv dex/solver.rs dex/mod.rs` makes sense. I don't know what I was thinking before 😛.

Reply:
The hint was in `solver::dex::solver` module path 😛